docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+>This feature is useful for other situations but doesn’t apply here.
+:::
+[[Return to Advanced Search]](#6-focus-on-the-right-file)
+ 
+### 2. Copy the string
+*You will only need the* <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> *so copy it without the quotation marks!*
+
+Like so:
+`989928F2-06D6-42F3-871A-353F07DEF969_LEGACY_HERO_BLOODMONEYSUIT_M_HPA2293_NAME_`
+<div id="multilangua"></div>
+<details>
+<summary>Multi-language</summary>
+:::info multi-language
+Try pasting your copied <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> in <Tooltip text={<strong>the search box</strong>} tooltip= "The one that appears when you search inside the 'Preview box'" />.
+
+You will see your <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> attached to multiple languages of your suit's name. 
+This shows that a specific <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> will always be the same in every other language `{array}` of a <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." />. 
+>This is relevant for later!
+
+[[Return to Adding More Languages]](#5-optionally-add-more-languages)
+:::
+</details>
+
+
+---
+
+## Pt. 3: **CONVERT TO DECIMAL**
+*We need to convert the* <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" /> *to a* <Tooltip text={<><strong>decimal number</strong><span className="icon-alert"></span></>} tooltip="A string converted into a numeral form" /> *to make it usable in the manifest file*.
+>That's what the <Tooltip text={<><strong>Text Tools tab</strong><span className="icon-alert"></span></>} tooltip="A tab in GlacierKit that provides tools for converting text and hashes for game modifications." /> is for.
+
+
+### 1. Go to Text Tools 
+- Go to the <Tooltip text={<strong>Text Tools</strong>} tooltip="A tab in GlacierKit that provides tools for converting text and hashes for game modifications." /> tab
+- Locate the <Tooltip text={<><strong>Localisation Hash Calculator</strong><span className="icon-alert"></span></>} tooltip= "A tool that converts your string into a format the manifest can use, generating both hex and decimal codes." /> box. 
+
+### 2. Convert your string
+- Paste your string inside that box
+- Copy *only* the converted <Tooltip text={<strong>decimal number</strong>} tooltip="A string converted into a numeral form" />. 
+
+>*We now have everything needed for the next part:*
+ >- a <Tooltip text={<strong>decimal form</strong>} tooltip="A string converted into a numeral form" /> of our <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" />
```

Comment:
The "decimal form" is in fact also a hash (because all a hash is is a piece of text calculated from another piece of text), which is confusing I know but people should probably be told about to avoid further naming confusion.

Ideally we would be calling these "localisation hashes" and the ones used to refer to game files "RuntimeIDs" or "RuntimeResourceIDs"; unfortunately RPKG tool took the name hash and applied it to literally everything possible, including calling game resources "hash files"